### PR TITLE
Implement Queue Expr

### PR DESCRIPTION
**Pull request by Dhiver**
I needed to enqueue packet to a userspace application with netfilter queue so I added the Queue expression.

You can now use the queue statements listed [here](https://wiki.nftables.org/wiki-nftables/index.php/Quick_reference-nftables_in_10_minutes#Queue).

### CODE REVIEW

**Comment by Dhiver** [comment]
I signed it!

File: expr/queue.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	"golang.org/x/sys/unix"
+)
+
+type QueueAttribute uint16
+
+type QueueFlag uint16
+
+// Possible QueueAttribute values
+const (
+	QueueAttributeNum   QueueAttribute = unix.NFTA_QUEUE_NUM
```
Comment (stapelberg) [inline expr/queue.go]: For consistency, these should just be `QueueNum`, `QueueTotal` and `QueueFlags` (without the `Attribute` part).

File: expr/queue.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
+package expr
```
Comment (stapelberg) [inline expr/queue.go]: Could you copy the copyright header into this file please?

File: expr/queue.go (hunk 1 above)
Comment (Dhiver) [reply expr/queue.go]: It's done.

File: expr/queue.go (hunk 2 above)
Comment (Dhiver) [reply expr/queue.go]: It's done.

**Comment by stapelberg** [comment]
Thanks for making the changes. Before I merge, could you squash your commits into one logical commit please, so that we can retain a clean git history?

Thanks!

**Comment by Dhiver** [comment]
It's done :)

**Comment by stapelberg** [comment]
Thanks!

**Pull request merged**